sdk/servicebus/azure-servicebus/azure/servicebus/exceptions.py

```diff
@@ -64,63 +85,97 @@ def _error_handler(error):
     return errors.ErrorAction(retry=True)
 
 
-def _create_servicebus_exception(logger, exception, handler):  # pylint: disable=too-many-statements
+def _handle_amqp_connection_error(logger, exception, handler):
```

Comment:
I copied and pasted the error description into our readme.
(I found our EH has something similar, so thanks again for the suggestion)

let me know how it looks to you.

Reply:
As mentioned OOB, I think this may be one of the more important changes that's gone in as far as materially improving QOL, for as silly as it may sound to harp on docs.  Wanted to echo my appreciation here as well.  Looks very solid, and will be a good base to evolve over time as needed.